recipes/prompt_toolkit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+    name: prompt_toolkit
+    version: "0.54"
+
+source:
+    fn: prompt_toolkit-0.54.tar.gz
+    url: https://pypi.python.org/packages/source/p/prompt_toolkit/prompt_toolkit-0.54.tar.gz
+    md5: c9b83f8ddc2d887aca78a10779aff623
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - pygments
+        - six >=1.9.0
+        - wcwidth
+
+test:
+    imports:
+        - prompt_toolkit
+        - prompt_toolkit.clipboard
+        - prompt_toolkit.contrib
+        - prompt_toolkit.contrib.completers
+        - prompt_toolkit.contrib.regular_languages
+        - prompt_toolkit.contrib.telnet
```

Comment:
Maybe we should send a patch upstream to return a more informative message here.  Fixing this now...